### PR TITLE
Feature/dynamic height

### PR DESCRIPTION
**Pull request by hossamghareeb**

### CODE REVIEW

File: DropDown/src/DropDown.swift [hunk 1]
```diff
@@ -477,9 +488,9 @@ extension DropDown {
 		xConstraint.constant = layout.x
 		yConstraint.constant = layout.y
 		widthConstraint.constant = layout.width
-		heightConstraint.constant = layout.visibleHeight
+		heightConstraint.constant = min(layout.visibleHeight, tableView.contentSize.height)
 
-		tableView.isScrollEnabled = layout.offscreenHeight > 0
+//		tableView.isScrollEnabled = layout.offscreenHeight > 0
```
Comment (dingua) [inline DropDown/src/DropDown.swift]: kill it :)

File: DropDown/src/DropDown.swift [hunk 2]
```diff
@@ -879,7 +892,7 @@ extension DropDown {
 
 	/// Returns the height needed to display all cells.
 	fileprivate var tableHeight: CGFloat {
-		return tableView.rowHeight * CGFloat(dataSource.count)
+		return tableView.estimatedRowHeight * CGFloat(dataSource.count)
```
Comment (dingua) [inline DropDown/src/DropDown.swift:885]: do you really need `tableHeight` if so, as your height is dynamic it will not reflect the real height.

Comment (hossamghareeb) [reply DropDown/src/DropDown.swift:885]: we need it as it works in fixed or dynamic height

File: DropDown/src/DropDown.swift (hunk 1 above)
Comment (hossamghareeb) [reply DropDown/src/DropDown.swift]: I would keep incase someone needed it later, to know where to change

**Pull request merged**